public/app/plugins/datasource/loki/languageUtils.ts

```diff
@@ -111,3 +113,56 @@ export function getLabelTypeFromFrame(labelKey: string, frame?: DataFrame, index
       return null;
   }
 }
+
+export const mapOpToAbstractOp: Record<AbstractLabelOperator, string> = {
+  [AbstractLabelOperator.Equal]: '=',
+  [AbstractLabelOperator.NotEqual]: '!=',
+  [AbstractLabelOperator.EqualRegEx]: '=~',
+  [AbstractLabelOperator.NotEqualRegEx]: '!~',
+};
+
+// eslint-disable-next-line @typescript-eslint/consistent-type-assertions
+export const mapAbstractOperatorsToOp = invert(mapOpToAbstractOp) as Record<string, AbstractLabelOperator>;
```

Comment:
Very clever! Honestly, I'm starting to dislike enums as they cannot be accessed by index, like it happens for LogLevel.